src/quo/theme.cljs

```diff
@@ -1,16 +1,25 @@
 (ns quo.theme
```

Comment:
Initially, I changed this to quo.context because this file now provides multiple contexts, not just the theme. However, the PR became [too big](https://github.com/user-attachments/assets/14491720-e96d-485f-85e8-e1a73d76d1f9). So better to do rename in another PR.